src/Symfony/Component/HttpFoundation/Session/Storage/Handler/MemcachedSessionHandler.php

```diff
@@ -77,7 +77,7 @@ protected function doRead(string $sessionId)
     #[\ReturnTypeWillChange]
     public function updateTimestamp($sessionId, $data)
     {
-        $this->memcached->touch($this->prefix.$sessionId, time() + (int) ($this->ttl ?? \ini_get('session.gc_maxlifetime')));
```

Comment:
I've read up how PHP's memcached extension interprets the values passed here: https://www.php.net/manual/en/memcached.expiration.php

tl;dr: If the passed integer is < 30 days in seconds, the value is considered to be a relative TTL. Otherwise, the value is assumed to be a UNIX timestamp.

My guess is that we added `time()` to be sure our relative TTL is never mistaken as a timestamp.

If we go back to relative TTLs again, I suggest we make sure our TTL is below that threshold and continue to ass `time()` otherwise.

Reply:
yes indeed https://github.com/memcached/memcached/blob/master/doc/protocol.txt#L139